src/taipy/gui_core/_context.py

```diff
@@ -143,7 +144,9 @@ def process_event(self, event: Event):
                         _GuiCoreContext._CORE_CHANGED_NAME, {"scenario": [x for x in sequence.parent_ids]}
                     )
             except Exception as e:
-                _warn(f"Sequence ({sequence.id if hasattr(sequence, 'id') else 'No_id'}) access raised an issue", e)
```

Comment:
a better fix would be to use event.entity_id instead of sequence.id if hasattr(sequence, 'id') else '<anonymous>'
No need to declare sequence before then

Reply:
Works for me